pkg/render/intrusion_detection.go

```diff
@@ -155,11 +190,33 @@ func (c *intrusionDetectionComponent) Objects() ([]client.Object, []client.Objec
 			c.intrusionDetectionPSPClusterRoleBinding())
 	}
 
+	if c.dpiDaemonSet {
+		c.intrusionDetectionComponentResources()
```

Comment:
Is this still needed here ? We are actually appending https://github.com/tigera/operator/pull/1474/files#diff-41a87360d53438a77f76d9f2a90345ded1d71cb682d4db44041ee2f16a78a504R216

Reply:
Yes, the resource requirements needs to be set before appending, as those values are needed for creating daemonset. I've added a comment in the code too.